LKML: https://lkml.org/lkml/2012/11/19/604
Git: http://git.kernel.org/?p=linux/kernel/git/sameo/mfd-2.6.git;a=commitdiff;h=aec17ea1ee78923f1287c18e9905c19e4dfd9c64


From: Samuel Ortiz <sameo@linux.intel.com>
Date: Fri, 9 Nov 2012 09:19:54 +0000 (+0100)
Subject: mfd: Explicitely include slab.h to rtsx
X-Git-Tag: mfd-3.8-1~106
X-Git-Url: http://git.kernel.org/?p=linux%2Fkernel%2Fgit%2Fsameo%2Fmfd-2.6.git;a=commitdiff_plain;h=aec17ea1ee78923f1287c18e9905c19e4dfd9c64

mfd: Explicitely include slab.h to rtsx

This fixes the following build error on some architectures (parisc at least):

drivers/mfd/rtsx_pcr.c: In function 'rtsx_pci_init_chip':
drivers/mfd/rtsx_pcr.c:985:2: error: implicit declaration of function 'kcalloc'
[-Werror=implicit-function-declaration]
drivers/mfd/rtsx_pcr.c:985:13: warning: assignment makes pointer from integer
without a cast [enabled by default]

Signed-off-by: Samuel Ortiz <sameo@linux.intel.com>
---

diff --git a/drivers/mfd/rtsx_pcr.c b/drivers/mfd/rtsx_pcr.c
index 56d4377..3a44efa 100644
--- a/drivers/mfd/rtsx_pcr.c
+++ b/drivers/mfd/rtsx_pcr.c
@@ -22,6 +22,7 @@
 
 #include <linux/pci.h>
 #include <linux/module.h>
+#include <linux/slab.h>
 #include <linux/dma-mapping.h>
 #include <linux/highmem.h>
 #include <linux/interrupt.h>
